### ISSUE TITLE
Make typealias for ComponentTypeAndStateByPerformerID

### ISSUE THREAD

**Issue by jsbean** [jsbean/DNM#154]
```Swift
typealias ComponentSpanData = [PerformerID: [String: Bool]]
```
?

**Comment by jsbean** [comment]
This gets passed around a lot between 

`DNMScoreViewModel`, `ComponentSpan`, `ComponentSpans`, then should also be in `ScoreView`, `PageView`, etc.

**Comment by jsbean** [comment]
Closed with 197352d72f2ffd7af1739c9879ac390dd3e70591
